tcf_core/settings/testing.py

```diff
@@ -0,0 +1,18 @@
+# pylint: disable=import-error,unused-wildcard-import,wildcard-import
+"""Django settings module for testing environment such as Heroku"""
```

Comment:
I feel like we've had this discussion somewhere before (don't remember where, so I'll say it again), but I think the name of this file ought to be something like `dev` or maybe `dev-build` for clarity at the expense of conciseness; `testing.py` sounds like a scratch file used for experimenting with the settings themselves.

Reply:
Bump in case you haven't checked my last comment: https://github.com/thecourseforum/theCourseForum2/pull/331#discussion_r544808854.
I can change it if you still think we should go with `dev` after reading this.

Reply:
Ah, knew it was somewhere. I see your point - I still feel like since I'm pretty sure the term "dev build" is pretty standard, we should go with that, but maybe something like `test-build` would work too? Just averse to the word `testing` specifically. Probably should get input from others - @j-alicia-long @jackliu612  ?

Reply:
I've thought more about this and found more reasons to go with `dev`:
1. Our default branch is `dev`.
2. The URLs are `dev.thecourseforum.com` and `thecourseforum-dev.herokuapp.com`.
3. The order of the files matches the flow of the source code integration:
![image](https://user-images.githubusercontent.com/7676354/106097636-abccf980-6105-11eb-8e33-fdd77c1a8281.png)

Hence, fixed in 489f15807ce4cd3dca935c52a8f000a06d08bf3f.